ex2_hal/adcs/equipment_handler/src/adcs_handler.c

```diff
@@ -3100,74 +3105,211 @@ ADCS_returnState ADCS_get_full_config(adcs_config *config) {
     float coef;
     ADCS_returnState state;
     state = adcs_telemetry(GET_FULL_CONFIG_ID, telemetry, 504);
+
     memcpy(&config->MTQ, &telemetry[0], 3);
     memcpy(&config->RW[0], &telemetry[3], 4);
     memcpy(&config->rate_gyro, &telemetry[7], 3);
     get_xyz(&config->rate_gyro.sensor_offset, &telemetry[10], 0.001);
     config->rate_gyro.rate_sensor_mult = telemetry[16];
+
     memcpy(&config->css, &telemetry[17], 10);
     coef = 0.01;
     for (int i = 0; i < 10; i++) {
-        config->css.rel_scale[i] = telemetry[27 + i] * coef;
+        config->css.rel_scale[i] = telemetry[27 + i] * coef; // Could switch css.rel_scale to integer?
     }
+
     config->css.threshold = telemetry[37];
+
     get_xyz(&config->cubesense.cam1_sense.mounting_angle, &telemetry[38], 0.01);
     config->cubesense.cam1_sense.detect_th = telemetry[44];
     config->cubesense.cam1_sense.auto_adjust = telemetry[45] & 1;
-    memcpy(&config->cubesense.cam1_sense.exposure_t, &telemetry[46], 2);
+
+    config->cubesense.cam1_sense.exposure_t = ((telemetry[47] << 8) | telemetry[46]); // WRONG >>> FIXED
+
     config->cubesense.cam1_sense.boresight_x = ((telemetry[49] << 8) | telemetry[48]) * coef;
     config->cubesense.cam1_sense.boresight_y = ((telemetry[51] << 8) | telemetry[50]) * coef;
     get_xyz(&config->cubesense.cam2_sense.mounting_angle, &telemetry[52], 0.01);
     config->cubesense.cam2_sense.detect_th = telemetry[58];
     config->cubesense.cam2_sense.auto_adjust = telemetry[59] & 1;
-    memcpy(&config->cubesense.cam2_sense.exposure_t, &telemetry[60], 2);
+    config->cubesense.cam2_sense.exposure_t = ((telemetry[61] << 8) | telemetry[60]); // WRONG >>> FIXED
```

Comment:
// WRONG >>> FIXED

This isn't a helpful comment. We know it's wrong, that's why this commit exists